entrypoint/versionbump/etcd_test.go

```diff
@@ -0,0 +1,68 @@
+//go:build etcd
+
+package versionbump
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/pkg/customflag"
+	"github.com/rancher/distros-test-framework/pkg/template"
+	"github.com/rancher/distros-test-framework/pkg/testcase"
+
+	. "github.com/onsi/ginkgo/v2"
+)
+
+var _ = Describe("VersionTemplate Upgrade:", func() {
+	It("Start Up with no issues", func() {
+		testcase.TestBuildCluster(GinkgoT())
+	})
+
+	It("Validate Node", func() {
+		testcase.TestNodeStatus(
+			assert.NodeAssertReadyStatus(),
+			nil)
+	})
+
+	It("Validate Pod", func() {
+		testcase.TestPodStatus(
+			assert.PodAssertRestart(),
+			assert.PodAssertReady(),
+			assert.PodAssertStatus())
+	})
+
+	It("Verifies bump version on product for etcd", func() {
+		cmd := "sudo journalctl -u k3s | grep 'etcd-version' ," + " k3s -v"
+		if cfg.Product == "rke2" {
+			cmd = "sudo /var/lib/rancher/rke2/bin/crictl -r unix:///run/k3s/containerd/containerd.sock images | grep etcd ," +
+				" rke2 -v"
+		}
+
+		template.VersionTemplate(template.VersionTestTemplate{
+			TestCombination: &template.RunCmd{
+				Run: []template.TestMap{
+					{
+						Cmd:                  cmd,
+						ExpectedValue:        template.TestMapTemplate.ExpectedValue,
```

Comment:
Can we change this to currentValue and other to upgradedValue? 
For non-upgrade scenarios current and upgradedValue would be same. If upgradedValue is not passed currentValue=upgradedValue.